packages/dto/auth/subscriptionForm/fromType.ts

```diff
@@ -0,0 +1,7 @@
+export enum FromTypeEnum {
+    INTRODUCTION = "INTRODUCTION",
+    RESEARCH = "RESEARCH",
```

Comment:
```suggestion
    OWN_SEARCH = "OWN_SEARCH",
```
Celui-là je suis pas convaincue mais `RESEARCH` ça m'évoque la recherche scientifique

Reply:
Ca veut pas dire grand chose je trouve. On parle de quoi, une recherche personnelle sur le navigateur comme "données sur les subventions des associations" avec google ?

Reply:
C'est ce que je comprends mais il faudrait voir avec Marion pour être sûrs

Reply:
Oui effectivement je suis d'accord @alice-telescoop avec tes idées. J'ai essayé de retranscrire les intitulés des champs mais effectivement pas super parlant.
Pour recherche, dans le front l'intitulé c'est "Via des recherches sur Internet", c'est plutôt moi qui ait mal traduit je pense, est-ce que "SEARCH_ENGINE" ça vous paraît plus cohérent ? @mxmeunier

Reply:
mais `SEARCH_ENGINE` me va aussi si `WEB` trop vaste pour vous